lang/syn/src/idl/file.rs

```diff
@@ -413,7 +413,28 @@ fn parse_ty_defs(ctx: &CrateContext) -> Result<Vec<IdlTypeDefinition>> {
                         })
                     })
                     .collect::<Result<Vec<IdlField>>>(),
-                syn::Fields::Unnamed(_) => return None,
+                syn::Fields::Unnamed(fields) => {
+                    let mut i = -1;
```

Comment:
`iter().enumerate().map(|(i, f) {...}` can be used instead of `i`

Reply:
Thanks, Improved.